conda/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+CONDA_BUILD_SYSROOT:
+  - /Applications/Xcode_13.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk       # [osx]
```

Comment:
This looks like it might break in the future with `macos-latest`.

Reply:
Yes, you are right. I can instead manually download specific MacOSX SDK's from https://github.com/phracker/MacOSX-SDKs (which seems to be a common practice elsewhere) to a specific location on the runner and use that instead.